tests/test_event.py

```diff
@@ -67,6 +68,36 @@ def test_init_no_end(self):
         event = Event('Lunch', start, timezone=TEST_TIMEZONE)
         self.assertEqual(event.end, start + 1 * hours)
 
+    def test_init_with_microseconds(self):
+        with self.assertLogs("gcsa.event", level="WARNING") as cm:
+            Event(
+                "Test",
+                start=datetime.datetime(
+                    year=1979, month=1, day=1, hour=1, minute=1, second=1, microsecond=1
+                ),
+            )
+            self.assertEqual(
+                cm.output,
+                [
+                    "WARNING:gcsa.event:Microseconds are used in start/end, but are not supported in Google Calendar API, and will be dropped on submission."
+                ],
+            )
+
+    def test_init_without_title(self):
+        with self.assertLogs("gcsa.event", level="WARNING") as cm:
+            Event(
+                "",
+                start=datetime.datetime(
+                    year=1979, month=1, day=1, hour=1, minute=1, second=1, microsecond=0
+                ),
+            )
+            self.assertEqual(
+                cm.output,
+                [
+                    "WARNING:gcsa.event:Summary is empty in 1979-01-01 01:01:01+01:00 - . Note that if the event is loaded from Google Calendar, its summary will be `None`"
```

Comment:
Microseconds should be 0 (i.e. "1979-01-01 01:01:01+00:00")

Reply:
Oh right, my bad, +01:00 is a timezone offset which is set to the timezone of your PC by default but it will fail in actions as they have UTC (I believe). Could you please set it explicitly for the `Event` (`timezone=TEST_TIMEZONE`, which is Fiji, so should translate to`"1979-01-01 01:01:01+12:00"`)?

Reply:
Ah, makes sense. Fixed.